src/sentry/models/sentryappcomponent.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import absolute_import
+
+import six
+import uuid
+
+from django.db import models
+
+from sentry.db.models import EncryptedJsonField, FlexibleForeignKey, Model
+
+
+def default_uuid():
+    return six.binary_type(uuid.uuid4())
+
+
+class SentryAppComponent(Model):
+    __core__ = True
+
+    uuid = models.CharField(max_length=64, default=default_uuid)
```

Comment:
Is there not a UNIQUE on this? Also, if it's strictly a UUID, with hyphens, that's 36 characters. Is there any reason this would be different?

We also have a UUIDField natively in our code base that you can use, that supports an actual uuid type which is much more efficient in Postgres.

Reply:
UNIQUE: there should be, thanks.
Hyphens: Was just trying to be consistent – I thought we used the non-hypen version everything for some reason.
UUIDField: Didn't realized that – would rather it be a actual UUID 👍